cmd/generate/vault.go

```diff
@@ -38,9 +38,14 @@ func newVaultClient(config vaultClientConfig) (*vaultapi.Client, error) {
 	return vaultClient, nil
 }
 
-func createVaultClientUsingOpsctl(ctx context.Context, gitHubToken, installation string) (*vaultapi.Client, error) {
+func createVaultClientUsingOpsctl(ctx context.Context, gitHubToken, user, installation string) (*vaultapi.Client, error) {
 	cmdArgs := []string{"opsctl", "create", "vaultconfig", "-i", installation, "-o", "json"}
 
+	if user != "" {
```

Comment:
Me being lazy right now, there is probably a nicer way to append here.

Reply:
I think you can do `cmdArgs = append(cmdArgs, "--user", user)`.